style.css

```diff
@@ -77,7 +77,6 @@ section {
   grid-template-rows: repeat(2, 1fr);
   grid-column-gap: 2rem;
   /* grid-row-gap: 2rem; */
```

Comment:
> Great Job so far.

- Please Kindly delete this commented line of code since it wasn't used. This will Ensure that your code is always clean and appealing.